wurst/dummy/DummyRecycler.wurst

```diff
@@ -143,6 +145,10 @@ init
 			DummyRecycler.angleQueues[i].enqueue(createDummy(boundMax, DUMMY_PLAYER, (i* ANGLE_DEGREE).asAngleDegrees()))
 
 @compiletime function generateDummyUnit()
+	let a = new AbilityDefinitionPermanentInvisibility(ABIL_ID_GEN.next())
```

Comment:
Is there some documentation for this behavior somewhere, in particular the `-1` values? Given blizzard's recent changes, I'm just a little wary of relying on undefined behaviour.

Reply:
I have tested it on the previous versions and on the current one, and it works. I have found this 'hack' a few years ago and I have been using it since then, could not find it now though. There is another way to achieve the same effect, but it will require changing acquire range after the unit creation using a native, since changing AR to 0 at compile time will reduce unit's attack range. It could be a fine solution if the dummy is used only through the recycler API, but it's ID is visible to everyone so it could be created in other places and then it's gonna bug.

Reply:
While I'm a little bit wary too, I think that the chance of Blizzard suddenly changing the effect of a slightly 'nonsence' value used in a single one, not frequently used and not causing any troubles ability is extremely low.

Reply:
Fine with me.

I would like to see thinking like this described in comments, but the quality there across the standard library is pretty variable, so I am not too fussed.